CHANGELOG.md

```diff
@@ -158,3 +158,7 @@
  ## [v2.13.0](https://github.com/BoltApp/bolt-magento2/releases/tag/2.13.0) 2020-08-19
  - Improved back-end components related to checkout experience.
  - Support for the Mageplaza plugin's shipping restriction
+
+ ## [v2.14.0](https://github.com/BoltApp/bolt-magento2/releases/tag/2.14.0) 2020-09-03
```

Comment:
Is this a tag date or the actual release date, which should be 2020-09-09?

Reply:
Good catch @danac-gs 
@lbliii This should be the target release date.

Reply:
@lbliii I've fixed that.